### ISSUE TITLE
Trying to only return 1 row from 'rows' returns list of list, but should probably return just a list.

### ISSUE THREAD

**Issue by ktr** [ktr/sxl#3]
I think when you try to grab 1 row (e.g., ''ws.rows[1]'' to grab the 1st row of a worksheet), it should simply return that row.

**Comment by jkyeung** [comment]
Well, `rows` is meant to be an iterator. Iterators cannot be accessed by arbitrary index, they can only be accessed one element at a time starting from the beginning.

**Comment by jkyeung** [comment]
Wait a minute. Are you just leaving *yourself* notes for planned fixes and new features?

You should use a better example than `ws.rows[1]` because that would be the second row, in Python. Which I guess is kinda-sorta the first "real" row **if** you have a header row and are not counting it. But for the sake of illustration, you should be saying something like "I'd like `ws.row[3]` to just return the fourth row in the sheet".

**Comment by ktr** [comment]
Yes, this one is tough. I also struggled with whether `ws.rows[0]` should return the first row or ws.rows[1] should. `ws.rows[0]` is definitely more pythonic, but it always seemed like more work than necessary to always remember to translate.

Right now `ws.rows[0]` doesn't return anything and `ws.rows[1]` returns the first row, but perhaps I will/should change that.
